src/main/java/Duke.java

```diff
@@ -1,10 +1,96 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
+
```

Comment:
Code seems very procedural and seems to lack of abstraction as the program execution instructions seems to be all within the main function of the Duke.java class.